extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -72,50 +68,75 @@ private function getWallMessageMock( array $params ) {
 				'isEdited',
 				'getLastEditSummary',
 				'getMetaTitle',
+				'load'
 			] )
 			->disableOriginalConstructor()
 			->getMock();
+		$wallMessageCounter = 0;
 
-		$wm->expects( $this->once() )
+		$wallMessageMock->expects( $this->at( $wallMessageCounter++ ) )
+			->method( 'load' );
+
+		$wallMessageMock->expects( $this->at( $wallMessageCounter++ ) )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this increment or decrement operator into a dedicated statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS881)